src/components/ReadingGoalPage/utils/validator.ts

```diff
@@ -29,9 +30,10 @@ interface ReadingGoalPayload {
 export const validateReadingGoalData = (
   chaptersData: ChaptersData,
   { type, pages, seconds, range }: ReadingGoalPayload,
+  mushafId: Mushaf = Mushaf.QCFV1,
```

Comment:
I think we should remove the default value from here since `mushafId` is required always passed.